packages/flutter_tools/lib/src/reporting/usage.dart

```diff
@@ -423,8 +423,7 @@ class _DefaultUsage implements Usage {
         isFirstRun ||
         // Display the welcome message if we are not on master, and if the
         // persistent tool state instructs that we should.
-        (!globals.flutterVersion.isMaster &&
-        (globals.persistentToolState.redisplayWelcomeMessage ?? true))) {
+        (globals.persistentToolState.redisplayWelcomeMessage ?? true)) {
```

Comment:
why did you remove the master check?

Reply:
This is the only place we're doing this - I think practically there isn't really a need to hide master since it should already be hidden on bots

Reply:
I thought this was an intentional usability improvement for devs on the master channel. It was added here: https://github.com/flutter/flutter/pull/43217/files. That having been said, I don't have a strong preference either way, before this change I just ignored the message.